ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -7044,6 +7044,13 @@ LM:
   - level_status
   - percentage_sensor
 
+LVLHL:
```

Comment:
I would suggest changing the name of this abstract type to something like LHLM (level high low monitoring) to be more consistent with the existing level monitoring abstract type, LM, and the general way abstract types are structured (control/monitoring tag on the end).

Reply:
@shambergoldstein - thank you, that makes a lot more sense. Have changed from LVLHL to LHLM.